change/@uifabric-charting-2020-03-06-11-00-41-user-v-sivsar-StackedAndMultiStackedbarAccesiblity.json

```diff
@@ -0,0 +1,9 @@
+{
+  "type": "patch",
+  "comment": "fixed accessiblity for the  statcked and multistacked bar chart",
```

Comment:
Please polish the commit message as these go out to our customers in release notes. Suggestion "StackedBarChart: added id for each callout and setting aria-labelledby accordingly"

Reply:
done